lib/v2/screens/profile_screens/guardians/guardians_tabs/interactor/viewmodels/guardians_events.dart

```diff
@@ -31,3 +22,30 @@ class OnAddGuardiansTapped extends GuardiansEvent {
   @override
   String toString() => 'OnAddGuardiansTapped ';
 }
+
+class OnAcceptGuardianTapped extends GuardiansEvent {
+  final String guardianAccount;
+
+  OnAcceptGuardianTapped(this.guardianAccount);
+
+  @override
+  String toString() => 'OnAcceptGuardianTapped : { OnAcceptGuardianTapped: $guardianAccount }';
```

Comment:
Wrong fromat in all this events (you repeat the event name instead print the value name).

`'OnAcceptGuardianTapped : { OnAcceptGuardianTapped: $guardianAccount }'`
must be:
`'OnAcceptGuardianTapped : { guardianAccount: $guardianAccount }'`